modules/ppcp-wc-gateway/src/Gateway/ProcessPaymentTrait.php

```diff
@@ -298,6 +298,12 @@ function ( Authorization $authorization ): bool {
 				);
 			}
 
+			$error_message = $error->getMessage();
+			if ( $error->issues() ) {
+				$error_message = $error->issues()[0]->issue . ' ' . $error->issues()[0]->description;
```

Comment:
and in other places (in PHP, ...) too then. `implode(array_map(.....))`

Reply:
Done! :)